src/dotnet-install.ps1

```diff
@@ -12,7 +12,7 @@
 .PARAMETER Channel
     Default: LTS
     Download from the Channel specified. Possible values:
-    - Current - most current release
+    - STS - standard term support SDK
```

Comment:
Some people are probably using "current" and they will not notice the change for a long time unless they check the documentation.

Should we add a warning to say "'Current' channel option is being deprecated. Use 'sts' instead."?

Reply:
done